source/styleEditor.js

```diff
@@ -1,3 +1,13 @@
 $(document).ready(function(){
-  // your code goes here.
+
+  $('form').on('submit', function(event){
+     event.preventDefault();
+   });
+
+  $("#style_editor").submit(function() {
```

Comment:
Rather than `.submit()`, I'd recommend sticking with `.on('submit', ` like you have with your general form event binding. This is more commonly practiced in the JS world.